generic_camera/canon/Dockerfile.aarch64

```diff
@@ -38,7 +38,34 @@ RUN wget -q https://repo-nexus.lsst.org/nexus/repository/ts_yum/releases/OpenSpl
   && pip3 install jupyterlab
 
 # All-Sky camera specific configuration
-RUN pip3 install gphoto2 Pillow
+RUN cd /tmp \
+  && git clone https://github.com/gphoto/libgphoto2.git \
+  && cd libgphoto2 \
+  && autoreconf --install --symlink \
+  && ./configure --prefix=/usr/local \
+  && make \
+  && make install \
+# Copy the .pc files so pkg-config can find them later on
+  && cp libgphoto2.pc /usr/share/pkgconfig \
+  && cp libgphoto2_port/libgphoto2_port.pc /usr/share/pkgconfig \
+  && echo "/usr/local/lib" > /etc/ld.so.conf.d/gphoto2.conf \
+  && echo "/usr/local/lib/libgphoto2/2.5.25.1" > /etc/ld.so.conf.d/gphoto2.conf \
+  && echo "/usr/local/lib/libgphoto2_port/0.12.0" > /etc/ld.so.conf.d/gphoto2.conf \
+  && ldconfig
+RUN cd /tmp \
+  && git clone https://github.com/gphoto/gphoto2.git \
+  && cd gphoto2 \
+  && autoreconf --install --symlink \
+  && ./configure --prefix=/usr/local \
+  && make \
+  && make install
+RUN cd /tmp \
+  && git clone https://github.com/jim-easterbrook/python-gphoto2.git \
+  && cd python-gphoto2 \
+  && python setup.py build_swig \
```

Comment:
this is calling `python` and everything else is calling `python3`... is this correct?

Reply:
Thanks for spotting that. It is correct, because a symlink between python3 and python gets created in line 16, but I'll modify it to python3 anyway for clarity.